docs/features/connections.md

```diff
@@ -1,19 +1,22 @@
 ---
-id: connections 
-title: 'Grow your network with our connections feature!'
-sidebar_label: 'Connections'
+id: connections
+title: 'Grow Your Network with Our Connections Feature!'
+sidebar_label: Connections
 keywords:
-  - connections 
+  - connections
 ---
+
 ## What are Connections?
-Connections on OpenSauced serve as a way to build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our "List" feature, and you come across several promising developers. What's next? With Connections, you can now send a request to connect, bringing these experts into your professional circle for easy communication and collaboration.
+
+Connections on OpenSauced serve as a way to build and nurture your professional network within the open source ecosystem. Imagine you're searching for experts in Artificial Intelligence using our [List](lists.md) feature, and you come across several promising developers. What's next? With Connections, you can now send a request to connect, bringing these experts into your professional circle for easy communication and collaboration.
 
 ![connections demo](../../static/gif/connections.gif)
 
 ## How Do Connections Work?
+
 - **Sending Connection Requests**: Once you identify users you'd like to connect with, you can send them a connection request.
 - **Acceptance**: Upon acceptance of your connection request, both parties become 1st-degree connections.
 - **Following Highlights**: After becoming 1st-degree connections, you can follow each other's highlights, keeping up-to-date with accomplishments, contributions, and other notable activities.
 - **Contact Information**: Post-acceptance, you'll have access to contact information to foster direct communication and ongoing dialogue.
```

Comment:
- When we follow a dev on OpenSauced and they follow us back, does it mean we are 1st-degree connection? And if we follow someone, although they don't follow us back, we can follow their highlights. If so, we don't need Connections feature to be a 1st-degree connection.
  
  Unless, we make it clear that 1st-degree connection means that they have access to send message for further collaboration communication.
- I think this line would be too strong for people who are concern with privacy:

  > **Contact Information**: Post-acceptance, *you'll have access to contact information* to foster direct communication and ongoing dialogue.

In my opinion, the steps here need to be updated as well, based on our experience, @BekahHW. 

When we tried this feature, if I recall, I sent you a request, but I didn't know if this request had been sent. And I didn't know if you accepted my request until I received an e-mail a moment later. There was delay in receiving the e-mail.

The concern here was if a user sent a request, and there is no notification or anything that telling them that the request has been sent, they might send multiple requests.

Reply:
Sending connections is kind of complicated right now. I am not sure how to create an issue for updating, but I'm going to make a note and we can revisit it in the future.

Reply:
Sure! Let's leave this comment open for our future reference.

Reply:
https://github.com/open-sauced/docs/issues/228